Update launcher-aiming-signoff.md

cjdrake42 opened the pull request:
Added an explanation for the needed precision of the motors and gave more insight into the speed of the motors.

cjdrake42 committed:
Update launcher-aiming-signoff.md

Added an explanation for the needed precision of the motors and gave more insight into the speed of the motors.

JTJones73 commented:
"and head on projectile sensing" change to "sensor"
"the ball" we need consistency and should probably call it "the target"
"C20: Launcher pitch and yaw must be controllable to a precision of within 0.6 degrees
C21: Time to move launcher into aiming position must be less than 0.08 s" consistency either use abbreviations or type out seconds in a section
" maximum processing time is estimated to take 0.1 ms" my algorithm isn't that good make it 100 ms
Schematic: Use voltage and ground nodes to make the schematic neater
"tor driver [4], and launch angle motor[5]." inconsistent spacing
"recommended by the site" what does this mean? + typo
"it is expected" I would probably avoid this wording in favor of "It has been determined that..."
cite the datasheet figures
"BOM" spell out bill of materials
Use IEEE citations

cjdrake42 committed:
Update launcher-aiming-signoff.md

Made appropriate changes based on feedback

cjdrake42 committed:
Add files via upload

Changed Electrical Schematic to have voltage and ground nodes to make it neater

cjdrake42 commented:
> "and head on projectile sensing" change to "sensor" "the ball" we need consistency and should probably call it "the target" "C20: Launcher pitch and yaw must be controllable to a precision of within 0.6 degrees C21: Time to move launcher into aiming position must be less than 0.08 s" consistency either use abbreviations or type out seconds in a section " maximum processing time is estimated to take 0.1 ms" my algorithm isn't that good make it 100 ms Schematic: Use voltage and ground nodes to make the schematic neater "tor driver [4], and launch angle motor[5]." inconsistent spacing "recommended by the site" what does this mean? + typo "it is expected" I would probably avoid this wording in favor of "It has been determined that..." cite the datasheet figures "BOM" spell out bill of materials Use IEEE citations

Made necessary changes

cjdrake42 committed:
Update launcher-aiming-signoff.md

Changed timing schedule for launcher

cjdrake42 committed:
Update launcher-aiming-signoff.md

Updated C21 calculations

cjdrake42 committed:
Update launcher-aiming-signoff.md

Fixed double repeating word

JesseTNRoberts reviewed (commented):
This looks like it was not created correctly. Please save all files and create a new pull request using the described process.

cjdrake42 committed:
Delete Documentation/Signoffs/launcher-aiming-signoff.md
